wordpress-mobile/gutenberg-mobile
# Try running UI tests on any updates to develop

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/4101

This PR follows up on the proposal in https://github.com/wordpress-mobile/gutenberg-mobile/issues/4101 to automatically run our full automated test suite on all changes to the `develop` branch. This is still just a proposal, so the idea of this PR is to try this idea out before we decide on it. If we're happy with it, we can merge this PR to put the proposal into practice.

⚠️ Before merging: If these changes work and are approved, we need to remove the testing branch (this PR's branch) from the config and set `post-to-slack` to `true`.

## To test

1. Ensure a new `Test iOS on Device - Full (On Updated Develop)` is seen below under CI checks
2. Ensure a new `Test Android on Device - Full (On Updated Develop)` is seen below under CI checks

Note: The two new workflows will only run on the `develop` branch once this PR is merged. They only appear on this PR because we're temporarily adding this PR's branch to the list of branches the workflow runs on.

PR submission checklist:
